src/main/java/core/basesyntax/service/operation/impl/OperationBalanceHandler.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.service.operation.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.operation.OperationHandler;
+
+public class OperationBalanceHandler implements OperationHandler {
```

Comment:
The class is missing a private field `operationDao` of type `Map<Fruit, Integer>` and a constructor to initialize it. Add the field and constructor to make the code work.